querydsl-apt/src/test/java/com/querydsl/apt/DateExtensionsTest.java

```diff
@@ -34,7 +34,7 @@
     private static final String packagePath = "src/test/java/com/querydsl/apt/";
 
     @Test
-    public void Handles_Date_Extensions_Correctly() throws IOException, InterruptedException {
+    public void handles_Date_Extensions_Correctly() throws IOException, InterruptedException {
```

Comment:
I see the following naming patterns we could use:
* camelCase
* camelCase_variant_in_lower_case_snake_case
* requirement_in_lowercase_snake_case

What do you think?